src/Client.php

```diff
@@ -244,4 +251,11 @@ public function getLogger(): \Psr\Log\LoggerInterface
 
         return $this->logger;
     }
+
+    public function enableDebugLogging(bool $enable = true): Client
```

Comment:
I had the same thought, but cos it's such a non-standard thing, didn't want to pollute the config with it which is currently all about connection details. What about adding it as a second constructor parameter, default `false` ?